src/app/lib/init-help/init-help.component.spec.ts

```diff
@@ -10,43 +14,25 @@ import {
   HelpInitializationService
 } from '../../public';
 
-import {
-  HelpWindowRef
-} from '../window-ref';
-
-class MockWindowRef {
-  public nativeWindow = {
-    BBHELP: false
-  };
+function fullSpyOnClass<T>(type: Type<T>): jasmine.SpyObj<T> {
+  return jasmine.createSpyObj(type.name, Object.keys(type.prototype));
 }
 
 describe('HelpInitComponent', () => {
```

Comment:
it's not entirely obvious to me why this component exists. all it does is load the widget with a hardcoded `bbhelp` config, which is a dummy config meant solely for local dev. i was going to delete it but say that it was exported and ppl are using it. again, i have no idea why. but i went ahead and corrected the tests not to care about the deps, which they probably shouldn't have cared about in the first place.